tests/unit/test_utils.py

```diff
@@ -186,3 +191,12 @@ def test_exit_of_context_manager_for_pager(self):
     def test_stdout(self, mock_binary_out):
         with self.stream_factory.get_stdout_stream():
             self.assertTrue(mock_binary_out.called)
+
+    def test_can_silence_io_error_from_pager(self):
+        self.popen.return_value = MockProcess()
+        try:
+            with self.assertRaises(RuntimeError):
```

Comment:
Why does this raise a `RuntimeError`? I would assume that this test only consist of the try,except,fail statement

Reply:
The context manager never yields since I throw an IOError which in turn raises a RuntimeError which covers the IOError if I don't ignore it. I couldn't find a great seam otherwise for testing this. Either I let the yield happen and loose my chance to throw an IOError since there is no real process to throw it from, or I throw the IOError and also get a RuntimeError.

Reply:
Ahh might be good to put a comment there then